sdk/cosmos/azure-cosmos-spark_3-1_2-12/src/test/scala/com/azure/cosmos/spark/CosmosRowConverterSpec.scala

```diff
@@ -50,6 +50,50 @@ class CosmosRowConverterSpec extends UnitSpec with BasicLoggingTrait {
     objectNode.get(colName2).asText() shouldEqual colVal2
   }
 
+  "basic spark row with various numeric types" should "translate to ObjectNode" in {
```

Comment:
nit: THere is another test already doing this, might as well just add the new types to it https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/cosmos/azure-cosmos-spark_3-1_2-12/src/test/scala/com/azure/cosmos/spark/CosmosRowConverterSpec.scala#L390-L408

Reply:
Good catch! Thanks